weldx/utility.py

```diff
@@ -423,6 +435,14 @@ def xr_interp_like(
     else:  # assume da2 to be dict-like
         sel_coords = da2
 
+    # store and strip pint units at this point, since the unit is lost during
+    # interpolation and because of some other conflicts. Unit is restored before
+    # returning the result.
+    units = None
+    if isinstance(da1.data, pint.Quantity):
+        units = da1.data.units
+        da1 = xr.DataArray(data=da1.data.magnitude, dims=da1.dims, coords=da1.coords)
```

Comment:
does `da1.data = da1.data.magnitude` work here?

Reply:
Doesn't work for some reason :( ---> Tests fail.